src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1062,7 +1068,121 @@ export default abstract class ComparisonStore
             return Promise.resolve([]);
         },
     });
+    @computed get alterationEnrichmentRowData(): AlterationEnrichmentRow[] {
+        if (
+            this.alterationsEnrichmentData.isComplete &&
+            this.alterationsEnrichmentAnalysisGroups.isComplete
+        )
+            return getAlterationRowData(
+                this.alterationsEnrichmentData.result!,
+                this.resultsViewStore
+                    ? this.resultsViewStore.hugoGeneSymbols
+                    : [],
+                this.alterationsEnrichmentAnalysisGroups.result!
+            );
+        return [];
+    }
+
+    @computed get genesOfInterestForPathwayMapper(): AlterationEnrichmentRow[] {
+        let i: number = 0;
+        let newGeneOfInterest: AlterationEnrichmentRow[] = [];
+        let newGene: string = '';
+        let newGenes: string[] = [];
+        console.log(this.alterationAndPathwaysFlag);
+        console.log('adsds');
+        for (i = 0; i < this.alterationAndPathwaysFlag.length; i++) {
+            if (
+                this.alterationAndPathwaysFlag[i].charCodeAt(0) !== 10 &&
+                this.alterationAndPathwaysFlag[i].charCodeAt(0) !== 32
+            ) {
+                newGene =
+                    newGene + this.alterationAndPathwaysFlag[i].toUpperCase();
+                console.log(newGene);
+            } else {
+                console.log(newGene);
+                if (newGene !== '') newGenes.push(newGene);
+                newGene = '';
+            }
+        }
+        if (newGene !== '') newGenes.push(newGene);
+        console.log(newGenes);
+        console.log(this.alterationEnrichmentRowData);
+        newGeneOfInterest = this.alterationEnrichmentRowData.filter(gene => {
+            return this.doesInclude(gene.hugoGeneSymbol, newGenes);
+        });
+        console.log(newGeneOfInterest);
+        return newGeneOfInterest;
+    }
+
+    doesInclude(gene: string, newGenes: string[]) {
+        let i: number = 0;
+        //console.log(gene);
+        for (i = 0; i < newGenes.length; i++) {
+            if (newGenes[i] === gene) {
+                console.log(gene);
+                return true;
+            }
+        }
+        return false;
+    }
+
+    @computed get maxFrequencedGenes(): AlterationEnrichmentRow[] {
+        if (
+            this.alterationsEnrichmentData.isComplete &&
+            this.alterationsEnrichmentAnalysisGroups.isComplete
+        ) {
+            console.log('adsad');
+            //  return [];
+            let MaxFrequencedGenes: AlterationEnrichmentRow[] = [];
```

Comment:
avoid ordinary variable names starting with an uppercase letter. something like this is more proper:
```suggestion
            let genesWithMaxFrequency: AlterationEnrichmentRow[] = [];
```